### PR TITLE
3 prepare for ganache

### LINKED ISSUE #3: Prepare payment service for development in Ganache
Gelato does not support developing locally with Ganache so we need a way to have the payment service working.

### PR DESCRIPTION
**Pull request by juanenrisley**
closes #3 

- [X] Prepare /safes endpoint for local
- [X] Prepare /transactions endpoint for local
- [X] Add docs
- [x] Add tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +18,31 @@ cp .env.example .env
...
 ```bash
-$ npm start
+npm start
 ```
 
 This command starts a local development server and serves a Swagger UI at the `/docs` endpoint for documentation and example usage.
+
+### Using Gelato Relay
+
+By default the service will use a local strategy for funding transactions with a funder account unless the `NODE_ENV` environment variable value is set to `production`, then Gelato Relay will be used instead. This is especially important for local development with Ganache network because of Gelato not supporting it.
```
Comment (llunaCreixent) [inline README.md:48]: Shouldn't we add the `NODE_ENV` variable in the .env too?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+name: Run tests
+
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
+      DOCKER_COMPOSE: docker-compose -f docker-compose.yml -f docker-compose.frontend.yml -p circles
```
Comment (llunaCreixent) [inline .github/workflows/tests.yml:10]: I saw this `docker-compose.frontend.yml` file also in `circles-core`, but it doesn't exist anywhere, right?

File: README.md [hunk 3]
```diff
@@ -18,15 +18,31 @@ cp .env.example .env
...
+FALLBACK_HANDLER_ADDRESS=
+MULTI_SENT_ADDRESS=
+MULTI_SEND_CALL_ONLY_ADDRESS=
+```
 
 ### Run
 
 ```bash
-$ npm start
+npm start
```
Comment (llunaCreixent) [inline README.md:41]: In the README it's also missing the documentation for tests. I mean including the `npm test`

File: .github/workflows/tests.yml (hunk 2 above)
Comment (juanenrisley) [reply .github/workflows/tests.yml:10]: Yeah you are right, nice catch!

File: README.md (hunk 1 above)
Comment (juanenrisley) [reply README.md:54]: done!

**Review by llunaCreixent — APPROVED** [review]
This looks great!

**Review by JacqueGM — APPROVED** [review]
great! LGTM

**Review by JacqueGM — APPROVED** [review]

**Review by llunaCreixent — APPROVED** [review]
it looks great!

**Pull request merged**